cfs-require-import.php

```diff
@@ -8,32 +8,50 @@
 Author URI: https://www.vanpattenmedia.com/
 */
 
-class CfsRequireImport {
+include_once( ABSPATH . 'wp-admin/includes/plugin.php' );
 
+class CfsRequireImport {
+	
 	function __construct() {
-		register_activation_hook( plugin_basename( __FILE__ ), array( $this, 'import_cfs_fields' ) );
+		
+		/* Register activation hook. */
```

Comment:
Very small formatting/style thing: for the single-line comments, use the `// comment` syntax instead of `/* comment */`!

Reply:
Ok. I'll fix that.